external/python37.BUILD

```diff
@@ -0,0 +1,32 @@
+package(default_visibility = ["//visibility:public"])
+
+licenses(["notice"])
+
+cc_library(
+    name = "python37",
+    srcs = select(
+        {
+            ":x86_mode": glob([
+                "lib/python3.7/config-3.7m-x86_64-linux-gnu/libpython3.7.so",
+            ]),
+            ":arm_mode": glob([
+                "lib/python3.7/config-3.7m-aarch64-linux-gnu/libpython3.7.so",
+            ]),
+        },
+        no_match_error = "Please Build with an ARM or Linux x86_64 platform",
+    ),
+    hdrs = glob([
+        "include/python3.7/*.h",
+    ]),
+    includes = ["include/python3.7"],
+)
+
+config_setting(
```

Comment:
Don't define custom config settings here. It's hard to maintain.
Use `//tools/platform:x86_mode` and `//tools/platform:aarch64_mode` instead.